endpoints/put.go

```diff
@@ -87,7 +87,12 @@ func NewPutHandler(backend backends.Backend, maxNumValues int, allowKeys bool) f
 			}
 
 			logrus.Debugf("Storing value: %s", toCache)
-			resps.Responses[i].UUID = uuid.NewV4().String()
+			u2, err := uuid.NewV4()
+			if err != nil {
+				http.Error(w, fmt.Sprintf("Error generating version 4 UUID"), http.StatusBadRequest)
```

Comment:
We should probably return `http.StatusInternalServerError` rather than `http.StatusBadRequest` because this error isn't caused due to a bad request.

Reply:
This makes sense

Reply:
Does it make sense to change it Hans?

Reply:
Yes, let's use internal server error here.

Reply:
Done